python/ray/air/_internal/util.py

```diff
@@ -17,3 +18,22 @@ def is_nan(value):
 
 def is_nan_or_inf(value):
     return is_nan(value) or np.isinf(value)
+
+
+def shorten_tb(tb, attr: str):
+    should_not_shorten = bool(int(os.environ.get("RAY_AIR_FULL_TRACEBACKS", "0")))
```

Comment:
are we documenting this env var somewhere?

Reply:
Not yet. I was wondering if we should keep this internal for now - I'll look for a good place to put it though. Maybe some kind of FAQ item